### ISSUE TITLE
The tar function reports error in the bastion server

### ISSUE THREAD

**Issue by YFan0019** [gigascience/gigadb-website#1990]
**Describe the bug**

**To Reproduce**
1. Go to bastion server to Dropbox user243
2. Run `$tar -xf Comparative_Data.tar.bz2`
3. See error
```
tar (child): lbzip2：cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```
The tar file is fine when I use my own Mac to extract files, may caused by some missing library in Centos

**Expected behavior**
The tar function can work well.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS

**Smartphone (please complete the following information):**

**Additional context**

**Comment by rija** [comment]
Hi @YFan0019,

I've installed the missing command on the bastion. Can you try again.

**Comment by YFan0019** [comment]
Hi @rija,

Thanks for updating, I can use tar function now.

**Comment by rija** [comment]
Fixed in PR #1996
